integrations/appboy/lib/index.js

```diff
@@ -55,20 +56,80 @@ Appboy.prototype.initialize = function() {
   } else if (options.datacenter === 'eu') {
     customEndpoint = 'https://sdk.fra-01.braze.eu/api/v3';
   }
+  return customEndpoint;
+}
+
+function isVersionTwo(options) {
+  return Number(options.version) === 2;
+}
+
+function getConfig(options) {
+  var config = {};
+  if (isVersionTwo(options)) {
+    // https://js.appboycdn.com/web-sdk/2.0/doc/module-appboy.html#.initialize
+    config = {
+      safariWebsitePushId: options.safariWebsitePushId,
+      enableHtmlInAppMessages: options.enableHtmlInAppMessages,
+      allowCrawlerActivity: options.allowCrawlerActivity,
+      doNotLoadFontAwesome: options.doNotLoadFontAwesome,
+      enableLogging: options.enableLogging,
+      localization: options.localization,
+      minimumIntervalBetweenTriggerActionsInSeconds:
+        Number(options.minimumIntervalBetweenTriggerActionsInSeconds) || 30,
+      openInAppMessagesInNewTab: options.openInAppMessagesInNewTab,
+      openNewsFeedCardsInNewTab: options.openNewsFeedCardsInNewTab,
+      requireExplicitInAppMessageDismissal:
+        options.requireExplicitInAppMessageDismissal,
+      serviceWorkerLocation: options.serviceWorkerLocation,
+      sessionTimeoutInSeconds: Number(options.sessionTimeoutInSeconds) || 30
+    };
+  } else {
+    var datacenterMappings = {
+      us: 'https://sdk.iad-01.braze.com',
+      us02: 'https://sdk.iad-02.braze.com',
+      us03: 'https://sdk.iad-03.braze.com',
+      eu: 'https://sdk.fra-01.braze.eu'
+    };
+    if (options.safariWebsitePushId)
+      config.safariWebsitePushId = options.safariWebsitePushId;
+    if (options.enableHtmlInAppMessages) config.enableHtmlInAppMessages = true;
+
+    // Setup custom endpoints
+    if (options.customEndpoint) {
+      var endpoint = options.customEndpoint;
+      var regex = new RegExp('^(http|https)://', 'i');
+      config.baseUrl =
+        (regex.test(endpoint) ? endpoint : 'https://' + endpoint) + '/api/v3';
+    } else {
+      config.baseUrl =
+        (datacenterMappings[options.datacenter] ||
+          'https://sdk.iad-01.braze.com') + '/api/v3';
```

Comment:
```suggestion
          datacenterMappings['us']) + '/api/v3';
```
A change in one place might be missed in another

Reply:
Hmm.. not sure what this was. I didn't change anything here, just moved it around.